.eslintrc.json

```diff
@@ -22,7 +22,7 @@
         "@typescript-eslint/ban-types": "off",
         "@typescript-eslint/no-empty-interface": "off",
         "@typescript-eslint/no-explicit-any": "warn",
-        "@typescript-eslint/no-unused-vars": "warn",
+        "@typescript-eslint/no-unused-vars": "off",
```

Comment:
disabled this rule because it is causing issues with some cases when using state